mimecast/plugin.spec.yaml

```
@@ -701,8 +701,8 @@ actions:
         - malicious
         - all
       url_to_filter:
-        title: URL filter
-        description: URL to filter results with
+        title: Filter
+        description: Reguler exprestion matching to filter on. e.g. `examp` will return only URLs with the letters exap in them
```

A lot of misspelling here. Please run `misspell` on the spec and help.md.

```suggestion
        description: Regular expression to filter on. e.g. `examp` will return only URLs with the letters examp in them
```